build/FileUtils.csx

```diff
@@ -3,6 +3,12 @@
 using System.Text.RegularExpressions;
 
 
+public static string WithWindowsSlashes(this string path)
```

Comment:
ooh loose extension methods 😀

Reply:
You mean loose in the sense that it is too generic working off the string datatype? 

Maybe this is better then ? 
```
string FileUtils.WithWindowsSlashes(string path)
```

Reply:
no i meant it's perfect for scripts

Reply:
Oh I see  👍  :)

Reply:
The best part of adding this feature was the ability to debug the build script 👍